### ISSUE TITLE
A feature to block/unblock all programs within a directory would be great.

### ISSUE THREAD

**Issue by prashant-pant** [pylorak/TinyWall#44]

**Comment by coopfidgets** [comment]
Any update? Should be a relatively quick fix

**Comment by ilgrank** [comment]
> Any update? Should be a relatively quick fix

It seems you missed what's clearly stated on the project main page. I'm quoting it for you: 

> **Upstream development is now largely inactive**, but this repository is provided for anyone who would like to submit their own improvements or fork the project.

Now @coopfidgets , since it is a relatively quick fix, why don't you submit a PR for it so that it can be integrated in the repo?

**Comment by 9ao9ai9ar** [comment]
Please close the issue yourself as it is a duplicate of https://github.com/pylorak/TinyWall/issues/4, https://github.com/pylorak/TinyWall/issues/35, https://github.com/pylorak/TinyWall/issues/44 and https://github.com/pylorak/TinyWall/issues/46, and upvote the OP on https://github.com/pylorak/TinyWall/issues/4 instead, where the author of TinyWall has also personally pitched in with the conversation.
